Improve bug report template description (#802)
ansible/ansible-lint: .github/ISSUE_TEMPLATE/bug_report.md

---
name: 🐛 Bug report

> about: Create a bug report, only if you can reproduce on master branch.

Reviewer: ```suggestion
about: Create a bug report (only if it's reproducible on the master branch)
```

Author: @gundalow What do you think? Which one fits better with Cambridge English? ;)

Author: @awcrosby What do you think? Comma or parentheses for the descriptions used on https://github.com/ansible/ansible-lint/issues/new/choose ?

> about: Create a bug report, only if you can reproduce on master branch.

Reviewer: This introduces a bit of inconsistency with a trailing dot. Other options don't have it.
